basic/html-tag/html-tags.md

```diff
@@ -0,0 +1,176 @@
+title: Introduction to HTML Tags
+slug: https://10xdev.codeparrot.ai/
+cover image: Upload the relevant image at https://hashnode.com/uploader and use the url
```

Comment:
Add the following between the --- block 


slug: introduction-to-html-tags
tags: html
cover: https://cdn.hashnode.com/res/hashnode/image/upload/v1704720765233/wr1xLBKVV.png?auto=format
domain: 10xdev.codeparrot.ai
saveAsDraft: false